ideas/122-sob-increase_platform_usage.md

```diff
@@ -0,0 +1,60 @@
+## Preamble
+
+    Idea: 122-sob-increase_platform_usage
+    Title: Open Bounty - Increase Platform Usage
+    Status: Draft
+    Created: 2018-04-04
+    Related: 
+
+## Summary
+
+To identify the successful use of a platform, it's important and imperative to be able to measure its usage. As the Open Bounty platform continues to grow and evolve, its important to be able to track its growth, and make data-driven decisions.
+
+One of the key objectives of Open Bounty is to increase the usage of the platform. The purpose of this swarm is to build the tools and means necessary to capture these metrics and be able to report on them on a regular basis.
```

Comment:
Maybe drop the first sentence?

Reply:
Just trying to give the swarm a purpose - think it might be useful for those not intimately involved.